packages/lesswrong/components/common/HomeLatestPosts.tsx

```diff
@@ -127,7 +125,7 @@ const HomeLatestPosts = ({classes}: {classes: ClassesType}) => {
   const limit = parseInt(query.limit) || defaultLimit;
 
   const now = useCurrentTime();
-  const dateCutoff = moment(now).tz(timezone).subtract(frontpageDaysAgoCutoffSetting.get(), 'days').format("YYYY-MM-DD");
+  const dateCutoff = moment(now).subtract(frontpageDaysAgoCutoffSetting.get(), 'days').startOf('hour').toISOString()
```

Comment:
Does `useCurrentTime` give us any guarantee about the timezone of the time it returns? Like, are we going to get an SSR mismatch if it's different between client/server?

Jim and I were actually looked at this briefly a few days ago and ATM there are some situations where we get SSR mismatches (I think it was logged out users without a timezone cookie).

Reply:
This will be handled in [this PR](https://github.com/ForumMagnum/ForumMagnum/pull/9204), which will make it so the first render matches the server timezone (to avoid SSR mismatch), and then it re-renders with the correct timezone. I did it there because there is some similar logic wrt to the actual time being different (due to caching) in that PR